spec/features/collections/collection_details_spec.rb

```diff
@@ -202,6 +202,22 @@
     end
   end
 
+  context 'when selecting a collection with a description with links' do
+    before :all do
+      load_page '/search/collection-details', focus: 'C1200230663-MMT_1', env: :sit, ac: true
+    end
+
+    it 'displays the external link' do
+      expect(page).to have_content('Greenland Ice Mapping Project (GIMP)')
+      # expect(page).to have_link("Greenland Ice Mapping Project (GIMP)", href: "http://nsidc.org/data/measures/gimp")
+      # find_link("Greenland Ice Mapping Project (GIMP)")[:target].should == 'blank'
+    end
+
+    it 'does not display a link for relative paths' do
```

Comment:
Remove empty test, or supply a check as described.

Reply:
Same comment/questions as above - I noted this commit as "WIP" and pushed it so other devs could pull my changes and debug the failing tests. Can other devs pull my changes if I just commit and don't push? Rookie question